packages/wonder-blocks-dropdown/components/multi-select.js

```diff
@@ -262,7 +271,9 @@ export default class MultiSelect extends React.Component<Props, State> {
     render() {
         const {
             alignment,
+            "aria-labelledby": ariaLabelledBy,
             disabled,
+            id,
```

Comment:
I think we should use `...restProps` or `...otherProps` to pass through props otherwise we'll continue adding additional things to pass through.  We can control what's accepted via `AriaProps` and/or some other shared props type.

Reply:
Great point, I'm going to work on that as part of this ticket: https://khanacademy.atlassian.net/browse/WB-580

That way I'll be revisiting all the cases where we need to use this approach!

Reply:
@kevinbarabash As you suggested, I added this functionality as part of WB-180 (#411)

Reply:
Awesome, thanks!